index.html

```diff
@@ -504,42 +494,86 @@ <h3>🤖 AI Agent Compatible</h3>
     });
 });
 
-// GitHub stars & forks
-(async function () {
-  try {
-    const res = await fetch('https://api.github.com/repos/edgeleap/diffsense');
-    if (!res.ok) return;
-    const data = await res.json();
-    document.getElementById('stars-count').textContent =
-      (data.stargazers_count ?? 0).toLocaleString();
-    document.getElementById('forks-count').textContent =
-      (data.forks_count ?? 0).toLocaleString();
-  } catch (e) {
-    // keep fallback numbers
-  }
-})();
+// -------- Shields.io SVG parsing with 30s client-side TTL --------
+const BADGE_CACHE_TTL_MS = 30 * 1000; // 30 seconds
+
+function getCachedBadge(key) {
+    try {
+        const raw = localStorage.getItem(key);
+        if (!raw) return null;
+        const parsed = JSON.parse(raw);
+        if (!parsed || typeof parsed !== 'object') return null;
+        const { value, ts } = parsed;
+        if (typeof ts !== 'number') return null;
+        if (Date.now() - ts > BADGE_CACHE_TTL_MS) {
+            // Stale: treat as miss (no stale value returned)
+            return null;
+        }
+        return value;
+    } catch {
+        return null;
+    }
+}
+
+function setCachedBadge(key, value) {
+    try {
+        localStorage.setItem(key, JSON.stringify({ value, ts: Date.now() }));
+    } catch {
+        // Ignore storage errors
+    }
+}
+
+async function getBadgeValue(badgeUrl, cacheKey) {
+    const cached = getCachedBadge(cacheKey);
+    if (cached !== null) return cached;
+
+    try {
+        const res = await fetch(badgeUrl, { cache: 'no-store' });
+        if (!res.ok) return null;
+
+        const svgText = await res.text();
+        const parser = new DOMParser();
+        const svgDoc = parser.parseFromString(svgText, 'image/svg+xml');
```

Comment:
is it parsing svg badge? I might read this wrong. maybe yours is the same as mine. in any case I published mine and it works now.